c_glib/README.md

```diff
@@ -142,6 +142,18 @@ $ meson compile -C c_glib.build
 $ sudo meson install -C c_glib.build
 ```
 
+> [!WARNING]
+>
+> When building GLib, it typically uses the Arrow library installed via brew. However, this can lead to build failures
+> if there are mismatches between the changes in Arrow's GLib and CPP libraries. To resolve this, you may need to
+> reference the Arrow CPP library built locally. In such cases, use the `-Darrow_cpp_build_dir`
+> and `-Darrow_cpp_build_type` options with the `meson setup` command to explicitly specify the library path.
+>
+> ```bash
+> meson setup c_glib.build c_glib -Dgtk_doc=true \
+>   -Darrow_cpp_build_dir=${arrow_cpp_build_dir} \
+>   -Darrow_cpp_build_type=${arrow_cpp_build_type}
```

Comment:
How about using `--cmake-prefix-path` instead?

```bash
meson setup c_glib.build c_glib --cmake-prefix-path=${arrow_cpp_install_prefix} -Dgtk_doc=true
```

Note that Apache Arrow C++ must be installed. (Executing only `cmake --build` doesn't work. Executing `cmake --install` is needed.)